TypeCobol.LanguageServer.Test/LSRTests/MultiCommentSyntaxColoring/input/MultiCommentSyntaxColoring.tlsp

```diff
@@ -28,7 +28,7 @@
     },
     {
       "category": 1,
-      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/MAYANJE/AppData/Local/Temp/1/tcbl/DVZZJCM58896372926830602866.cee\",\"diagnostics\":[{\"range\":{\"start\":{\"line\":30,\"character\":6},\"end\":{\"line\":30,\"character\":6}},\"severity\":1,\"code\":\"44\",\"source\":\"Page\",\"message\":\"Invalid use of '%'. Should be used with \\\"%<< ... (linebreak) %>>\\\" for MultiLine Comment or \\\"%<<< ... (linebreak) %>>>\\\" for Formalized Comment\"},{\"range\":{\"start\":{\"line\":33,\"character\":6},\"end\":{\"line\":33,\"character\":6}},\"severity\":1,\"code\":\"44\",\"source\":\"Page\",\"message\":\"Invalid use of '%'. Should be used with \\\"%<< ... (linebreak) %>>\\\" for MultiLine Comment or \\\"%<<< ... (linebreak) %>>>\\\" for Formalized Comment\"},{\"range\":{\"start\":{\"line\":31,\"character\":55},\"end\":{\"line\":31,\"character\":60}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : Symbol MyData is not referenced\"},{\"range\":{\"start\":{\"line\":31,\"character\":21},\"end\":{\"line\":31,\"character\":60}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : Function not found 'TCOZDATE.CurrentDate'  output(?)\"}]}}"
+      "message": "{\"jsonrpc\":\"2.0\",\"method\":\"textDocument/publishDiagnostics\",\"params\":{\"uri\":\"file:///C:/Users/MAYANJE/AppData/Local/Temp/1/tcbl/DVZZJCM58896372926830602866.cee\",\"diagnostics\":[{\"range\":{\"start\":{\"line\":30,\"character\":6},\"end\":{\"line\":30,\"character\":6}},\"severity\":1,\"code\":\"44\",\"source\":\"Page\",\"message\":\"Invalid use of '%'. Should be used with \\\"%<< ... (linebreak) %>>\\\" for MultiLine Comment or \\\"%<<< ... (linebreak) %>>>\\\" for Formalized Comment\"},{\"range\":{\"start\":{\"line\":33,\"character\":6},\"end\":{\"line\":33,\"character\":6}},\"severity\":1,\"code\":\"44\",\"source\":\"Page\",\"message\":\"Invalid use of '%'. Should be used with \\\"%<< ... (linebreak) %>>\\\" for MultiLine Comment or \\\"%<<< ... (linebreak) %>>>\\\" for Formalized Comment\"},{\"range\":{\"start\":{\"line\":31,\"character\":55},\"end\":{\"line\":31,\"character\":60}},\"severity\":1,\"code\":\"30\",\"source\":\"Page\",\"message\":\"Semantic error: Symbol MyData is not referenced\"},{\"range\":{\"start\":{\"line\":31,\"character\":21},\"end\":{\"line\":31,\"character\":60}},\"severity\":1,\"code\":\"27\",\"source\":\"Find the syntax diagram describing the statement in error in the language reference\",\"message\":\"Syntax error : Function not found 'TCOZDATE.CurrentDate'  output(?)\"}]}}"
```

Comment:
The last part `Function not found 'TCOZDATE.CurrentDate'` looks like another semantic error that is currently reported as a syntax error. Maybe we should update it too in this issue.

Reply:
done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/4b5e1d0af58323248efbd75657d4b554eb7db081